io.github.jeffshee.Hidamari.json

```diff
@@ -0,0 +1,240 @@
+{
+    "app-id": "io.github.jeffshee.Hidamari",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "hidamari",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--filesystem=xdg-config/hidamari",
+        "--filesystem=xdg-config/autostart",
+        "--own-name=io.github.jeffshee.Hidamari.player",
+        "--own-name=io.github.jeffshee.Hidamari.server",
```

Comment:
```suggestion
```

`io.github.jeffshee.Hidamari.*` is granted by default

Reply:
Thanks!